apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -249,6 +255,8 @@ function main() {
           pairingAllowed: s.pairing_allowed,
           numStudents: s.students.length,
           code: s.code,
+          course_id: s.course_id,
+          script_id: s.script ? s.script.id : null,
```

Comment:
Nit: why not `courseId` / `scriptId` here?  Consistency with the existing Angular?